sdk/digitaltwins/digital-twins-core/README.md

```diff
@@ -138,9 +197,7 @@ For more detailed instructions on how to enable logs, you can look at the [@azur
 
 ## Next steps
 
-Please take a look at the
-[samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/digitaltwins/digital-twins-core/samples)
-directory for detailed examples on how to use this library.
+Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/digitaltwins/digital-twins-core/samples) directory for detailed examples that demonstrate how to use the client libraries.
```

Comment:
IIRC, there was some talk that there should be no samples for this package. If this is still true, please remove this part. If that is false, please make sure those samples also use `@azure/identity`.

Reply:
Let's keep the samples. We are using `@azure/identity` in the samples, and #14679 converts those samples to TypeScript and migrates to samples workflow v2.